DRIVERS/ETH_16Z077/DRIVER/men_16z077_eth.c

```diff
@@ -2414,10 +2414,8 @@ int men_16z077_probe( CHAMELEON_UNIT_T *chu )
 	}
 
 	/* set up timer to poll for link state changes */
-	init_timer(&np->timer);
 	np->timer.expires 	= jiffies + CONFIG_HZ / 2;
-	np->timer.data 		= (unsigned long)dev;
-	np->timer.function 	= z77_timerfunc;
+	timer_setup(&np->timer, (void *)(struct timer_list *)z77_timerfunc, 0);
```

Comment:
timer_setup is not present in the kernel 3.6.11

Reply:
Thanks for the hint. I'll let it apply for kernel version higher than 4.15.